includes/Modules/Analytics_4.php

```diff
@@ -1048,7 +1048,12 @@ protected function create_data_request( Data_Request $data ) {
 			case 'GET:accounts':
 				return $this->get_service( 'analyticsadmin' )->accounts->listAccounts();
 			case 'GET:account-summaries':
-				return $this->get_service( 'analyticsadmin' )->accountSummaries->listAccountSummaries( array( 'pageSize' => 200 ) );
+				return $this->get_service( 'analyticsadmin' )->accountSummaries->listAccountSummaries(
+					array(
+						'pageSize'  => 200,
+						'pageToken' => $data['pageToken'] ?? '',
```

Comment:
Does this break if `null` is passed as a `pageToken`? If not, then the `??` isn't necessary here since `$data` is a `Data_Request` which implements array access, and returns any undefined keys as `null` 👍 See `Data_RequestTest::test_array_access`